pkg/reporter/summary.go

```diff
@@ -667,6 +691,17 @@ func (r *summaryReporter) packageVulnerabilityRiskText(pkg *models.Package) stri
 	return WhiteBgText(" Unknown ")
 }
 
+func (r *summaryReporter) packageManifestRelativePath(pkg *models.Package, currentWorkingDirectory string) (string, error) {
+	fullPath := pkg.Manifest.Path
```

Comment:
Check that it is a valid local path using `os.Stat` and if not, return error early?